src/app.js

```diff
@@ -0,0 +1,22 @@
+import pkg from '@slack/bolt';
+import dotenv from 'dotenv';
+
+dotenv.config(); 
+
+const { App } = pkg;
+
+const app = new App({
+  token: process.env.SLACK_BOT_TOKEN,
+  appToken: process.env.SLACK_APP_TOKEN, 
+  socketMode: true, 
+});
+
+app.command('/bonus', async ({ command, ack, respond }) => {
+  await ack(); // Acknowledge the command
+  await respond(`Hello, <@${command.user_id}>! 👋`);
+});
+
+(async () => {
+  await app.start(); 
+  console.log('⚡️ Bolt app is running in Socket Mode!');
+})();
```

Comment:
While this gets around the "no top-level `await`" restriction, self-executing methods are a little ugly.
You could:
- Use the Promise [`.then()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then) API:
  ```typescript
  app.start().then(() => {
    console.log('⚡️ Bolt app is running in Socket Mode!');
  });
  ```
- Upgrade to Node 22 which allows top-level `await` (IIRC)